cves/kernel/CVE-2020-27170.yml

```diff
@@ -55,7 +55,7 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: This entry focuses on an issue dealing with specific subsystem within the linux kernel. A process known as "speculative execution" (where the cpu executes code before it needs to) did not handle certain loads properly, mainly surrounding applications performing pointer operations. This allowed an attack where certain memory that unprivileged users should not have access to could access certain kernel memory by essentially going out-of-bounds of memory with a specific load.
```

Comment:
This line is pretty long, might be nice to put it onto multiple lines

Reply:
Addressed